ZuluContent/Items/Weapons/Ranged/Bow.cs

```diff
@@ -9,23 +9,23 @@ public class Bow : BaseRanged
 
         public override Type AmmoType => typeof(Arrow);
 
-        public override Item Ammo { get; } = new Arrow();
+        public override Item Ammo => new Arrow();
```

Comment:
Seems like this is the only necessary change from the auto-property init to a lamba function. 

the difference between `public override Item Ammo { get; } = new Arrow();` and `public override Item Ammo => new Arrow();` is that with the `=> new Arrow();` lambda we're creating a new instance every access, whereas `{ get; } = new Arrow();` sets the property only once, and every access will return the same instance.

The code using the property expects a new instance of `Arrow()` to be spawned on every access so after the 1st Arrow is consumed it causes an exception.

The rest of the code like `public override int DefaultStrengthReq { get; } = 20;` is fine to use the auto-property initializer as it's effectively used as an overridden constant, any code using it is not expecting a new instance every access. Same goes for the enum properties too.

Reply:
Alright I removed the unneccessary ones :) thanks for explaining